android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java

```diff
@@ -371,6 +371,7 @@ public void onMethodCall(@NonNull MethodCall call, @NonNull Result result) {
             case "presentPaywallIfNeeded":
                 final String requiredEntitlementIdentifier = call.argument("requiredEntitlementIdentifier");
                 presentPaywall(result, requiredEntitlementIdentifier);
+                break;
```

Comment:
We probably would want to also add some tests for these but didn't want to delay fixing this... Definitely we should improve our coverage in the hybrids.

Reply:
oh wow, yes. Nice find